### DISCUSSION TITLE
Question about bitrate

### DISCUSSION THREAD

**Discussion by xqdd** [alvr-org/ALVR#1135] [Q&A] [answered]
I had been tested usb3.0(2Gbps) and wifi6(1200Mbps), but the performance of both of them are the same, why?
> My test result: max bitrate is about 430Mbps, and average bitrate is 300Mbps

Is there anyone exceeded this result? Could you please talk about your configuration or hardware devices, thanks a lot.

**Comment by xDShot** [comment, marked as answer]
PC is connected to router via Ethernet cable?

You can also determine if pc performance or network trafficking is bottleneck in dashboard monitoring.

**Reply by xqdd** [reply]
Well, let me answer it by myself, it depent on content's size(300Mbps is enough for transfer 8k stream data) and graphics card's hevc(NVENC for Nvida) video encode ability quest2's video decode ability.
[Refer To FAQ from moonlight](https://github.com/moonlight-stream/moonlight-docs/wiki/Frequently-Asked-Questions#why-doesnt-the-bitrate-slider-go-beyond-150-mbps)
